lib/schemes/flat.js

```diff
@@ -6,5 +6,15 @@ module.exports = {
         var naming = bemNaming(options.naming);
 
         return naming.stringify(entity) + '.' + tech;
+    },
+    parse: function(path, options) {
+        options || (options = {});
+        var naming = bemNaming(options.naming),
+            splittedPath = path.split('.');
+
+        return {
```

Comment:
Object like `{entity: {block, elem, mod}, tech: ?String}` has no name atm so it's `BemUnnamedObject`.
You are free to call it somehow.